src/services/ivaDatabaseService.ts

```diff
@@ -0,0 +1,66 @@
+import DynamoDB, { DocumentClient } from "aws-sdk/clients/dynamodb";
```

Comment:
There may be a legitimate reason, i'm just curious why this has been called ivaDatabaseService instead of something more consistent with what is already there, like ivaDefectDAO - as this is the database layer?

Reply:
The Data Access Object pattern isn't often seen outside of legacy Java applications and its misuse in the existing service is an anti-pattern that needs to be removed eventually, probably when that endpoint gets rewritten. Notice the lack of mapping code in this particular file, it only sends requests and returns responses from the database.

Reply:
Not sure I follow, is that not what the defectsDAO getAll method is doing, simply sending the request and returning the result?